### PR TITLE
Update to freedesktop 19.08

### PR DESCRIPTION
**Pull request by Erick555**
Fixes https://github.com/flathub/tv.kodi.Kodi/issues/67
Fixes https://github.com/flathub/tv.kodi.Kodi/issues/70
Fixes https://github.com/flathub/tv.kodi.Kodi/pull/61

### CODE REVIEW

**Comment by Erick555** [comment]
bot, build

**Comment by Erick555** [comment]
bot, build

**Comment by Erick555** [comment]
@wsnipex please test

File: tv.kodi.Kodi.yml [hunk 1]
```diff
@@ -27,18 +27,18 @@ command: kodi
...
   - deps/avahi.json
   - deps/libical.json
   - deps/bluez.json
   - deps/fmt.json
-  - deps/fribidi.json
   - deps/fstrcmp.json
   - deps/libass.json
   - deps/libcdio.json
   - deps/lzo2.json
+  - deps/libtirpc.json
```
Comment (Marko10-000) [inline tv.kodi.Kodi.yml]: Why is libtirpc added?

File: tv.kodi.Kodi.yml [hunk 2]
```diff
@@ -27,18 +27,18 @@ command: kodi
 rename-desktop-file: kodi.desktop
 rename-icon: kodi
 modules:
-  - shared-modules/udev/udev-175.json
+  - shared-modules/intltool/intltool-0.51.json
```
[the remark this answers is not in the document]
Comment (Erick555) [reply tv.kodi.Kodi.yml]: It's needed to build avahi (was dropped from Sdk), see [commit description](https://github.com/flathub/tv.kodi.Kodi/pull/69/commits/2fdbfdbcb85479113503e2cbea0bca68e8376c5b).

File: tv.kodi.Kodi.yml (hunk 1 above)
Comment (Erick555) [reply tv.kodi.Kodi.yml]: It's needed by samba after glibc dropped rpc headers, see [commit description](https://github.com/flathub/tv.kodi.Kodi/pull/69/commits/60b557d118ec764e13a63401d7dec4d5b5f2d65c).

**Comment by Erick555** [comment]
@Marko10-000 every commit has description why it's needed.

**Comment by Erick555** [comment]
ping @Marko10-000 @wsnipex

**Comment by Erick555** [comment]
bot, build

**Comment by Erick555** [comment]
@tilosp @comparascope @wsnipex @Marko10-000 I know that kodi developers lost interest in maintaining flatpak but I believe merging updates like this one is still worth doing rather than letting it rot.

**Comment by tilosp** [comment]
@Erick555 i would merge this pr if i could
but i have no push access to this repo

**Comment by Erick555** [comment]
@tilosp when I asked about this on irc, kodi developers were ok with passing maintenance of kodi flatpak to someone else. Would you accept such opportunity?

**Pull request merged**